cli/cli.ts

```diff
@@ -5452,6 +5476,18 @@ export function getCodeSnippets(fileName: string, md: string): CodeSnippet[] {
     }
     let snippets = getSnippets(md);
     const codeSnippets = snippets.filter(snip => !snip.ignore && !!supported[snip.type]);
+    const jres = snippets.filter(snip => snip.type === "jres")[0];
+
+    let extraFiles: Map<string> = null;
+
+    if (jres) {
+        extraFiles = {
```

Comment:
Maybe worth a helper somewhere to take jres.code string and return this object? Seems like it pops up in quite a few places

Probably not worth doing right now just for a minor amount of redundancy, and easy to do later just by searching for `emitTilemapsFromJRes` refs